src/test/three-components/TextureUtils-spec.ts

```diff
@@ -30,16 +30,14 @@ const HDR_EQUI_URL = assetPath('spruit_sunrise_2k.hdr');
 
 suite('TextureUtils', () => {
   let textureUtils: TextureUtils;
-  let renderer: WebGLRenderer;
+  const renderer = new WebGLRenderer({canvas});
```

Comment:
We should leave a note here, because `WebGLRenderer` is stateful and can lead to unexpected results in tests depending on the order they are run in down the road.

Reply:
Added.